storage/google/cloud/storage/bucket.py

```diff
@@ -1118,8 +1120,12 @@ def copy_blob(
             _target_object=new_blob,
         )
 
-        if not preserve_acl:
-            new_blob.acl.save(acl={}, client=client)
+        if preserve_acl is not None:
+            warnings.warn(
+                "The 'preserve_acl' argument is deprecated. Please update the bucket's ACL manually.",
```

Comment:
This isn't "deprecation":  we are changing the behavior, and emitting a warning.  If we *are* going to deprecate the argument, then we should preserve the existing behavior, and emit an explicit deprecation, e.g.:

```python
        if preserve_acl is not None:
            warnings.warn(
                "The 'preserve_acl' argument is deprecated.  For forward compatibility, "
                 "please update the blob's ACL manually.",
                 DeprecationWarning,
                 stacklevel=2,
            )
        if not preserve_acl and preserve_acl is not None:
            new_blob.acl.save(acl={}, client=client)
```

Reply:
If we do that then, the warning will always be emitted. I think the change makes sense.  Otherwise we can do a soft deprecation and only mark it in documentation.